capstone-frontend/src/components/CreateClub.js

```diff
@@ -0,0 +1,22 @@
+import React, { Component } from 'react';
+import '../App.css';
+
+class CreateClub extends Component {
+  render() {
+    return (
+      <form action="https://8080-0b34ed39-12e2-4bb0-83f0-3edbd4365bbd.us-east1.cloudshell.dev/clubs" method="post">
```

Comment:
What's our standard for these backend links? We probably will have to change them on our local machines every time they're cloned so we don't have a diff in this file every time

Reply:
I think we should change them to the production backend link. 

In this scenario, it should be ```"/api/clubs"```

Reply:
fixed!